Packages/vcs/Lib/VTKPlots.py

```diff
@@ -415,11 +417,12 @@ def plot(self,data1,data2,template,gtype,gname,bg,*args,**kargs):
             self._renderers[(None,None,None)]=ren
         else:
             ren = self._renderers[(None,None,None)]
-        vcs2vtk.genTextActor(ren,to=to,tt=tt)
+        returned["vtk_backend_text_actors"] = vcs2vtk.genTextActor(ren,to=to,tt=tt)
```

Comment:
of course not! @chaosphere2112 when we do the doc for widgets we should also doc these since you're using them as well.